backend/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM node:slim
```

I would suggest to use node 12 alpine, it's 1/5 the size of the slim image
```suggestion
FROM node:12-alpine
```

---

Yep probably better since I don't need we need any of the additional packages